src/libraries/System.Private.CoreLib/src/System/Buffer.Unix.cs

```diff
@@ -15,5 +15,6 @@ public static partial class Buffer
 #else
         private const nuint MemmoveNativeThreshold = 2048;
 #endif
+        internal const nuint ZeroMemoryNativeThreshold = 768;
```

Comment:
I would expect the average break-even point to be higher given that the vectorized implementation.